hydra-access-controls/spec/unit/visibility_spec.rb

```diff
@@ -11,6 +55,18 @@ class MockParent < ActiveFedora::Base
     include VisibilityOverride
   end
 
+  class Foo < ActiveFedora::Base
+    include Hydra::AccessControls::Permissions
+  end
+
+  describe do
+    subject { Foo.new }
+
+    include_examples "set visibility to X and then all others", Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE
```

Comment:
Do we need to make these shared examples since they aren't shared with anything else?  This seems like more indirection that is called for.